src/atoms/Size/__tests__/Size.web.test.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import View from '../../View';
+import Size from '../index';
+
+describe('<Size />', () => {
+  it('renders a child by applying height, width, maxHeight, maxWidth, minWidth, minHeight', () => {
+    const { container } = renderWithTheme(
+      <Size
+        height="100%"
+        width="10px"
+        maxHeight="20px"
+        maxWidth="20px"
+        minHeight="5px"
+        minWidth="5px"
+      >
+        <View />
+      </Size>,
+    );
+    expect(container).toMatchSnapshot();
+  });
+  it('renders a child by applying height, width, maxHeight, maxWidth, minWidth, minHeight in numeric values', () => {
+    const { container } = renderWithTheme(
+      <Size height="100" width="10" maxHeight="20" maxWidth="20" minHeight="5" minWidth="5">
```

Comment:
```suggestion
      <Size height={100} width={10} maxHeight={20} maxWidth={20} minHeight={5} minWidth={5}>
```
When you want to use scale you need to pass numbers and not strings

Reply:
oops, how did I miss this